packages/shopify-codemod/test/fixtures/split-return-assignments/multi.output.js

```diff
@@ -0,0 +1,14 @@
+foo(() => {
+  console.log(bar);
+  this.sho = this.doo;
+  this.doo = this.zoo;
+  this.zoo = 0;
+  return this.sho;
+});
+
+foo(() => {
+  foo();
+  foo = bar;
+  bar = baz;
```

Comment:
⚠️ this is not equivalent to `return foo = bar = baz;`. In that case, `foo` will equal whatever `baz` is. In this case, it will equal whatever `bar` is. We should just move the assignment up altogether rather than splitting it up.